Source/bmalloc/libpas/src/libpas/pas_probabilistic_guard_malloc_allocator.h

```diff
@@ -42,11 +42,11 @@ struct pas_pgm_storage {
     uintptr_t start_of_data_pages;
 
     /*
-     * These parameter below rely on page sizes being less than 65536.
+     * This parameter relies on page sizes being less than 65536.
      * I am not aware of any platforms using more than this at the moment.
      */
     uint16_t mem_to_waste;
-    uint16_t page_size;
+    bool freed;
```

Comment:
Enum allocations take up 4 bytes instead of 1

```
#include <stdbool.h>
#include <stdio.h>
int main() {
	enum AllocationState { Allocated, Freed };
	printf("sizeof(AllocationState) %lu\n", sizeof(enum AllocationState)); // 4 bytes
	printf("sizeof(bool) %lu\n", sizeof(bool)); // 1 byte
}
```

Reply:
Also, `enum class` is only available in C++.  Please ignore.